netpyne_ui/netpyne_geppetto.py

```diff
@@ -340,17 +340,15 @@ def _prepare_simulation_files(self, experiment: model.Experiment = None, use_pre
 
     def _prepare_batch_files(self, experiment: model.Experiment) -> str:
         # param path must be split: [ {'label': ['synMechParams', 'AMPA', 'gmax']} ]
-        # TODO: how do we handle array index? ['array', '[0]'] or ['array', '0']?
         exp =  copy.deepcopy(experiment)
         exp.params = [p for p in exp.params if p.mapsTo != '']
 
         for param in exp.params:
             if param.type == "range":
                 param.values = list(np.arange(param.min, param.max, param.step))
```

Comment:
this code is basically the same as above. What about extracting it?

Reply:
yeah I have a todo in `_create_trials` to generalize this logic, wanted to wait until it became clearer what the actual common logic will be, will extract it